src/Umbraco.Web.UI.Client/src/packages/tiptap/extensions/core/rich-text-essentials.tiptap-api.ts

```diff
@@ -14,13 +14,7 @@ export class UmbTiptapRichTextEssentialsExtensionApi extends UmbTiptapExtensionA
 
 	getTiptapExtensions = () => [
 		StarterKit,
-		Placeholder.configure({
-			placeholder: ({ node }) => {
-				return this.#localize.term(
-					node.type.name === 'heading' ? 'placeholders_rteHeading' : 'placeholders_rteParagraph',
-				);
-			},
-		}),
+		Placeholder.configure({ placeholder: this.#localize.term('placeholders_rteParagraph') }),
```

Comment:
This section gives a merge conflict to v15/dev. What is the reason for not showing its own text for headings?